### PR TITLE
retain logs for only 30 days

### PR DESCRIPTION
**Pull request by BernardZhao**
plus a helm chart bump

### CODE REVIEW

File: kubernetes/loki.yaml [hunk 1]
```diff
@@ -4,10 +4,9 @@ apiVersion: policy/v1beta1
 kind: PodSecurityPolicy
 metadata:
   name: loki
-  namespace: app-loki
```
Comment (ja5087) [inline kubernetes/loki.yaml:7]: is this important?

**Review by ja5087 — APPROVED** [review]

File: kubernetes/loki.yaml (hunk 1 above)
Comment (BernardZhao) [reply kubernetes/loki.yaml:7]: Honestly not sure, this seems just like the result of bumping the chart

**Review by nikhiljha — APPROVED** [review]
Unrelated: could we run the helm compilation in a makefile?

**Comment by ja5087** [comment]
> 
> 
> Unrelated: could we run the helm compilation in a makefile?

It is: https://github.com/ocf/loki/blob/9121e73adf5134b6baf63521364c54438be58093/Makefile#L11

**Comment by nikhiljha** [comment]
@ja5087 As in, create the kubernetes folder in the makefile and otherwise don't check it into git.

**Pull request merged**